src/Profiles/ANTPLUS_Profiles.h

```diff
@@ -4,5 +4,6 @@
 #include <Profiles/BicycleSpeed/ANTPLUS_BicycleSpeedProfile.h>
 #include <Profiles/HeartRate/ANTPLUS_HeartRateProfile.h>
 #include <Profiles/Environment/ANTPLUS_EnvironmentProfile.h>
+#include <Profiles/Lev/ANTPLUS_LevProfile.h>
```

Comment:
Looks like you have some capitalization differences here, looks like travis caught them
https://travis-ci.org/cujomalainey/antplus-arduino/jobs/544454378
(another joy is OSX and windows are case insensitive about paths but linux isn't)

Reply:
Done